### PR TITLE
Added versioncontrol.md

### PR DESCRIPTION
**Pull request by arunkarthicknm**

### CODE REVIEW

File: content/en/versioncontrol.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Definition Template
```
Comment (CathPag) [inline content/en/versioncontrol.md:2]: Please add the term here

File: content/en/versioncontrol.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+status: Feedback Appreciated
+category: Technology
+---
+
+## What it is
+Source control (or version control) is the practice of tracking and managing changes to code. To explain further, it is a system that records changes to a file or set of files over time so that you can recall specific versions later. 
+
+## Problem it addresses
+If you are a graphic or web designer (literally everyone who writes code) and want to keep every version of an image or layout, a Version Control System (VCS) is a very wise thing to use. It allows you to revert selected files back to a previous state, revert the entire project back to a previous state, compare changes over time, see who last modified something that might be causing a problem, who introduced an issue and when, and more.
```
Comment (CathPag) [inline content/en/versioncontrol.md:11]: This is not the problem, you are already explaining how it helps which goes in the next section :)  The problem it addresses should talk about what it was like before there was version control. Ideally, you don't even mention the word "version control" in this section. 

Also, graphic or web designers generally don't code. Are you referring to front-end developers? There are VCS for designers but I think we should focus on developers in this glossary.

**Review by CathPag — CHANGES_REQUESTED** [review]
This is a great start, @arunkarthicknm! Added two comments.

File: content/en/versioncontrol.md (hunk 2 above)
Comment (arunkarthicknm) [reply content/en/versioncontrol.md:11]: Sure, I'll change it :)

**Comment by arunkarthicknm** [comment]
@CathPag Will you please review it? :)

**Comment by arunkarthicknm** [comment]
@JasonMorgan Yeah sir, sure.

**Review by JasonMorgan — APPROVED** [review]

**Review by CathPag — APPROVED** [review]
Looks great!

**Comment by arunkarthicknm** [comment]
Thank you @JasonMorgan @CathPag !!❤️

**Pull request merged**